ci/build.yml

```diff
@@ -70,6 +70,47 @@ jobs:
         trigger_sha: '$(trigger_sha)'
     - template: report-end.yml
 
+- job: Linux_scala_2_13
```

Comment:
this is bad, this is going to fail for older releases. I’m not quite sure how to fix that unfortunately without starting to parse semvers in bash.